scout/server/blueprints/panels/templates/panels/panel.html

```diff
@@ -76,7 +76,7 @@
       </li>
       <li class="list-group-item">
         <label for="date">Date</label>
-        <span class="float-end" id="date">{{ panel.date.date() }}</span>
+        <span class="float-end" id="date">created:{{panel._id.generation_time.strftime('%Y-%m-%d')}} {% if panel.hidden %} deleted:{{ panel.date.date()}} {% endif %}</span>
```

Comment:
How about dropping "created:" entirely and perhaps adding tooltips with a few words about the date?

Reply:
I've added version date as you suggested. I think it's self explanatory, no?